product_supplierinfo_barcode/__manifest__.py

```diff
@@ -3,7 +3,7 @@
 {
     "name": "Barcode in supplier pricelist",
     "summary": "Add a barcode to supplier pricelist items",
-    "version": "12.0.1.1.0",
+    "version": "14.0.1.1.0",
```

Comment:
Should be 14.0.1.0.0

Reply:
hello @rousseldenis it would be nice if you give approval here.